lab_2/main.py

```diff
@@ -53,7 +95,20 @@ def calculate_distance(unknown_text_vector: list, known_text_vector: list) -> fl
     :param unknown_text_vector: vector for unknown text
     :param known_text_vector: vector for known text
     """
-    pass
+    if not (isinstance(unknown_text_vector, list) and isinstance(known_text_vector, list)):
+        return None
+
+    for num in unknown_text_vector:
+        if not (isinstance(num, (int, float))):
+            return None
+    for num in known_text_vector:
+        if not (isinstance(num, (int, float))):
+            return None
+    distance = 0
+    for index, _ in enumerate(unknown_text_vector):
+        distance += ((unknown_text_vector[index] - known_text_vector[index]) ** 2)
```

Comment:
Why not use the actual element and use `unknown_text_vector[index]`? You can easily access it with the `enumerate`.